pkg/otlp/testdata/metrics/allconfig.yaml

```diff
@@ -15,5 +15,7 @@ otlp_config:
       mode: counters
       send_count_sum_metrics: true
       send_aggregation_metrics: true
+  health_check:
+    port: 13134
```

Comment:
We have so far aligned with the configuration schema from upstream for the `debug` section (`logging` exporter), the `receiver` section (`otlp` receiver) and the `metrics` section (equivalent to `datadog` exporter's `metrics` section). 

The `health_check` extension uses `endpoint` instead of `port` for this, I think we should use that here instead of `port` to continue being aligned